lib/modulesync/cli.rb

```diff
@@ -1,54 +1,53 @@
-      class_option :namespace, :aliases => '-n', :desc => 'Remote github namespace (user or organization) to clone from and push to. Defaults to puppetlabs', :default => 'puppetlabs'
+      class_option :project_root, :aliases => '-c', :desc => 'The directory that contains a list of managed modules, file templates, and default values for template variables.', :default => Dir.pwd
+      class_option :namespace, :aliases => '-n', :desc => 'Remote github namespace (user or organization) to clone from and push to. Defaults to puppetlabs'
       class_option :filter, :aliases => '-f', :desc => 'A regular expression to filter repositories to update.'
-      class_option :branch, :aliases => '-b', :desc => 'Branch name to make the changes in. Defaults to master.', :default => 'master'
+      class_option :branch, :aliases => '-b', :desc => 'Branch name to make the changes in. Defaults to master.'
 
       desc 'update', 'Update the modules in managed_modules.yml'
+      option :modules_dir, :aliases => '-w', :desc => 'The directory where the modules should be cloned and synced. Defaults to ./modules'
       option :message, :aliases => '-m', :desc => 'Commit message to apply to updated modules. Required unless running in noop mode.'
-      option :configs, :aliases => '-c', :desc => 'The local directory or remote repository to define the list of managed modules, the file templates, and the default values for template variables.'
       option :remote_branch, :aliases => '-r', :desc => 'Remote branch name to push the changes to. Defaults to the branch name.'
-      option :amend, :type => :boolean, :desc => 'Amend previous commit', :default => false
-      option :force, :type => :boolean, :desc => 'Force push amended commit', :default => false
-      option :noop, :type => :boolean, :desc => 'No-op mode', :default => false
-      option :offline, :type => :boolean, :desc => 'Do not run any Git commands. Allows the user to manage Git outside of ModuleSync.', :default => false
-      option :bump, :type => :boolean, :desc => 'Bump module version to the next minor', :default => false
-      option :changelog, :type => :boolean, :desc => 'Update CHANGELOG.md if version was bumped', :default => false
-      option :tag, :type => :boolean, :desc => 'Git tag with the current module version', :defalut => false
+      option :amend, :type => :boolean, :desc => 'Amend previous commit'
+      option :force, :type => :boolean, :desc => 'Force push amended commit'
+      option :noop, :type => :boolean, :desc => 'No-op mode'
+      option :offline, :type => :boolean, :desc => 'Do not run any Git commands. Allows the user to manage Git outside of ModuleSync.'
+      option :bump, :type => :boolean, :desc => 'Bump module version to the next minor'
+      option :changelog, :type => :boolean, :desc => 'Update CHANGELOG.md if version was bumped'
+      option :tag, :type => :boolean, :desc => 'Git tag with the current module version'
```

Comment:
oh good question. i believe so, or at least they behave falsey so they could be `nil`. i'll double check